opcua_tools/nodeset_generator.py

```diff
@@ -91,19 +93,23 @@ def create_header_xml(
             namespaceheader += "<Uri>{}</Uri>\n".format(n)
         namespaceheader += "</NamespaceUris>\n"
 
+    model_uri = namespaces[serialize_namespace]
+    required_models = create_required_models(models, model_uri)
+
     return """<?xml version="1.0" encoding="utf-8"?>
 <UANodeSet LastModified="{}" {}>
 {}
 <Models>
-    <Model ModelUri="{}" PublicationDate="{}" Version="1.0.0"></Model>
+    <Model ModelUri="{}" PublicationDate="{}" Version="1.0.0">{}</Model>

```

Comment:
@olavpe I have one more idea that I didn't include in the changeset. As you can see above, the `Version` attribute is hardcoded. We'll have full parsed info about used models from now on - how about using the actual value instead of having it hardcoded? Quite the same thing could be done with the `PublicationDate` - usually, it is set to the current date and time by Model Builder but we could also extract that information from a model itself from now on. What do you think? Does the proposed solution have any disadvantages?

Reply:
As discussed, we'll go for setting version from the `UAModel` object, instead of hardcoded `1.0.0`. If the version attribute in a model is missing, then we go for the default: `1.0.0`.